src-plugins/itkDataImage/readers/itkDCMTKDataImageReader.cpp

```diff
@@ -357,117 +357,104 @@ bool itkDCMTKDataImageReader::readInformation (const QStringList& paths)
+        //StudyDate
+        //StudyTime
+
+        // SERIES
+        //SeriesID
+        updateMetaData(medData, medMetaDataKeys::SeriesDicomID.key(),     d->io->GetSeriesID().c_str());
+        updateMetaData(medData, medMetaDataKeys::SeriesNumber.key(),      d->io->GetSeriesNumber().c_str());
+        updateMetaData(medData, medMetaDataKeys::Modality.key(),          d->io->GetModality().c_str());
+        updateMetaData(medData, medMetaDataKeys::Performer.key(),         d->io->GetPerformingPhysicianName().c_str());
+        updateMetaData(medData, medMetaDataKeys::Report.key(), "");
+        updateMetaData(medData, medMetaDataKeys::Protocol.key(),          d->io->GetProtocolName().c_str());
+        updateMetaData(medData, medMetaDataKeys::SeriesDescription.key(), d->io->GetSeriesDescription().c_str());
+        //SeriesDate
+        //SeriesTime
+        //SeriesThumbnail
+
+        // IMAGE
+        //SOPInstanceUID
+        updateMetaData(medData, medMetaDataKeys::Columns.key(),         d->io->GetColumns().c_str());
+        updateMetaData(medData, medMetaDataKeys::Rows.key(),            d->io->GetRows().c_str());
+        //Dimensions
+        //NumberOfDimensions
+        updateMetaData(medData, medMetaDataKeys::Orientation.key(),     d->io->GetOrientation().c_str());
+        //Origin
+        updateMetaData(medData, medMetaDataKeys::SliceThickness.key(),  d->io->GetSliceThickness().c_str());
+        //ImportationDate
+        updateMetaData(medData, medMetaDataKeys::AcquisitionDate.key(), d->io->GetAcquisitionDate().c_str());
+        //AcquisitionTime
+        updateMetaData(medData, medMetaDataKeys::Comments.key(),        d->io->GetAcquisitionComments().c_str());
+
+        QStringList filePaths;
+        for (unsigned int i=0; i<d->io->GetOrderedFileNames().size(); i++)
+        {
+            if (d->io->GetOrderedFileNames()[i] != "")
+            {
+                filePaths << d->io->GetOrderedFileNames()[i].c_str();
+            }
+        }
+        if (filePaths.count() != 0)
+        {
+            medData->addMetaData(medMetaDataKeys::FilePaths.key(), filePaths);
+        }
 
+        updateMetaData(medData, medMetaDataKeys::Status.key(),          d->io->GetPatientStatus().c_str());
+        updateMetaData(medData, medMetaDataKeys::SequenceName.key(),    d->io->GetSequenceName().c_str());
+        //Size
+        //VolumeUID
+        //Spacing
+        //XSpacing
+        //YSpacing
+        //ZSpacing
+        //NumberOfComponents
+        //ComponentType
+        //PixelType
+        //medDataType
+        //PreferredDataReader
+        //ImageID
+        //ThumbnailPath
     }
     else {
-      qDebug() << "Unsupported pixel type";
-      return false;
+        qDebug() << "Unsupported pixel type";
+        return false;
     }
-    
+
     return true;
   }
 
+void itkDCMTKDataImageReader::updateMetaData(medAbstractData* medData, QString metakey, QString value)
```

Comment:
This function has been added to homogenize the way metadata are saved in `itkDCMTKDataImageReader::readInformation` above.